common/models/user.js

```diff
@@ -269,6 +269,14 @@ module.exports = function(User) {
    */
 
   User.logout = function(tokenId, fn) {
+    if(!tokenId) {
+      process.nextTick(function() {
+        var err = new Error('access token is required to logout');
+        err.status = 400;
+        fn(err);
+      });
+      return;
+    }
```

Comment:
Should https://github.com/strongloop/loopback/blob/fix/logout-without-token/common/models/user.js#L286 throw an error with statusCode set to 404 or 400 too?